Earthfile

```diff
@@ -416,7 +416,7 @@ base-image:
     IF [ "$VARIANT" = "standard" ]
         DO +PROVIDER_INSTALL -PROVIDER_KAIROS_BRANCH=${PROVIDER_KAIROS_BRANCH}
 
-        DO +INSTALL_K3S
+        DO +INSTALL_K3S --FLAVOR=$FLAVOR
```

Comment:
This passes the arg but I dont think the command is getting it as its not declared there

Reply:
Yes, earthly is strange. This is a `DO` so it gets everything propagated automatically. Don't ask me more... :D

Reply:
https://docs.earthly.dev/docs/earthfile#description-21

Reply:
ok found it. `FLAVOR` is a global (at the top of the file). That's why specifying it or not doesn't make any difference. Since we do that in other places, I will make it a "local" ARG for consistency.

Reply:
Interesting, so it inherits the caller context and args but it also creates its own context. weird.